src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -90,11 +91,17 @@ public abstract class InteractiveEvaluator : IInteractiveEvaluator, ICurrentWork
             _commandsFactory = commandsFactory;
             _commands = commands;
 
-            var hostPath = interactiveHostPath;
-            _interactiveHost = new InteractiveHost(replType, hostPath, initialWorkingDirectory);
-            _interactiveHost.ProcessStarting += ProcessStarting;
-
+            // The following settings will apply when the REPL starts without .rsp file.
+            // They are discarded once the REPL is reset.
+            ReferenceSearchPaths = ImmutableArray<string>.Empty;
```

Comment:
What is the expected behavior here? repl will start without .rsp file when noconfig option is used. Then if a reset is done these would be discarded? If, yes then why? What would be used instead?